### DISCUSSION TITLE
Minor 2.2.0

### DISCUSSION THREAD

**Discussion by XFox111** [XFox111/PasswordGeneratorExtension#95] [General]
> ## ❄️ Patch Tuesday update. 2022 Wrapped
> This release is a part of our new initiative!
From now on we are starting to roll out updates on every first Tuesday of the month, which will include bugfixes, security and dependency updates to keep the project's security and stability up to date!
> This is the last update this year. Next update will come out in February 2023

## What's new
- Dependency updates and security patches
- Updated Brazilian Portuguese locale (by @maisondasilva)
- Fixed "Add shortcut to context menu" checkbox tooltip trigger (#81)
- Added a Holiday Special ;)

## Known issues
- Vulnerabilities CVE-2022-37603 and CVE-2022-37599 remain unresolved for some dependencies as there's no compatible patched version yet

### From extension webstore (recommended)
- [Google Chrome Webstore](https://chrome.google.com/webstore/detail/jnjobgjobffgmgfnkpkjfjkkfhfikmfl)
- [Microsoft Edge Add-ons Webstore](https://microsoftedge.microsoft.com/addons/detail/manimdhobjbkfpeeehlhhneookiokpbj)
- [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/easy-password-generator/)
- [GitHub Releases](https://github.com/xfox111/PasswordGeneratorExtension/releases/latest)

Note that the version published on these webstores can differ from this release
### Sideloading (for testing purposes only)
1. Download attached archive and unpack it
2. Enable Developers mode on your browser extensions page
3. Click the "Load unpacked" button and navigate to the extension root folder (contains `manifest.json`)
4. Done!

*On Firefox you should open manifest file instead of extension's folder

**Note:** If you delete the extension folder it will disappear from your browser

_Sideloaded extensions do not replace officially installed ones_

<hr /><em>This discussion was created from the release <a href='https://github.com/XFox111/PasswordGeneratorExtension/releases/tag/v2.2.0'>Minor 2.2.0</a>.</em>

**Comment by maisondasilva** [comment]
Thanks @XFox111
